greenbone/openvas-scanner
# Fix: result len of aes{128,256}_gcm_{encrypt,decrypt}_auth

The result len when using
- aes_128_gcm_encrypt_auth
- aes_128_gcm_decrypt_auth
was incorrect. It was computing it by:
`((datalen / 16) + 1) * 16`
although it should be datalen.

To verify it there a is a new test in:
- nasl/tests/test_crypt_data_aes.nasl

And to verify it the execution Makefile is simplified and added to
github workflows.

To manually verify it go to `nasl/tests` and execute `make check`.
